package.json

```diff
@@ -109,5 +109,8 @@
     "tslib": "^2.4.1",
     "typescript": "^5.2.2",
     "typescript-plugin-css-modules": "^5.0.0"
+  },
+  "dependencies": {
+    "esno": "^0.17.0"
```

Comment:
This should be moved to `devDependencies` no?

Reply:
Yes, good catch, on it